python/kserve/kserve/protocol/grpc/server.py

```diff
@@ -12,35 +12,34 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
+import asyncio
 import logging
+import multiprocessing
 from concurrent import futures
 
-from kserve.grpc import grpc_predict_v2_pb2_grpc
-from kserve.grpc.interceptors import LoggingInterceptor
-from kserve.grpc.servicer import InferenceServicer
-from kserve.handlers.dataplane import DataPlane
-from kserve.handlers.model_repository_extension import ModelRepositoryExtension
+from . import grpc_predict_v2_pb2_grpc
+from .interceptors import LoggingInterceptor
+from .servicer import InferenceServicer
+from kserve.protocol.dataplane import DataPlane
+from kserve.protocol.model_repository_extension import ModelRepositoryExtension
 
 from grpc import aio
 
 
-MAX_GRPC_MESSAGE_LENGTH = 8388608
+MAX_GRPC_MESSAGE_LENGTH = -1
```

Comment:
Can we  keep it unlimited, what's the downside?

Reply:
i guess it just is user problem to handle but otherwise we limit them. i think they can override it in their own app anyways if they want